### DISCUSSION TITLE
MIB2Q patching problem

### DISCUSSION THREAD

**Discussion by Dagu3311** [Mr-MIBonk/M.I.B._More-Incredible-Bash#35] [Q&A] [answered]
Hello,

I have a problem with the patch.
I have the software MHI2Q_ER_AUG22_P5092 with MU-1329 on my MIB2Q
I followed the instructions but when patching, I got the following error at the end: 
**_**Patch can only be applied to stock firmware.**_**

I think someone has already done something since the car was converted from US to EU.

Anything else I can try?

Sorry for my English, I'm translating from German to English.
I would appreciate an answer.
[LOG.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9599773/LOG.log)

**Comment by Dagu3311** [comment, marked as answer]
Hello,

I reinstalled the software and now everything worked!
Thank you for the great tool!

I still have another question. Is it possible to update the Gracenote with MIB?

**Comment by harman-f** [comment]
Sure, GraceNote can be undated to latest version available.
check mibsolution

**Comment by luigi1980** [comment]
Hello.... I have the same problem as you, can you help me understand how you solved ? Thank you
